src/esbmc/bmc.cpp

```diff
@@ -137,6 +139,24 @@ void bmct::error_trace(
   msg.result(oss.str());
 }
 
+void printAliveMessage(const messaget &msg)
+{
+  std::stringstream ss;
```

Comment:
Couldn't you put this inside the while? Then you wouldn't need to call `s.str("")` later.